File: 파이썬베이스/정규표현식/반복/게으른수량자.py
import re


text = "This offer is not available to customers living in <b>AK</b> and <b>HI</b>"

# 방법 1
print(re.findall(r"<[Bb]>.*<\/[Bb]>", text))
# 게으른 수량자
print(re.findall(r"<[Bb]>.*?<\/[Bb]>", text))

File: 자스베이스/정규표현식/반복/게으른수량자.js
const text =
  "This offer is not available to customers living in <b>AK</b> and <b>HI</b>";

// 방법 1
console.log(text.match(/<[Bb]>.*<\/[Bb]>/g));
// 게으른 수량자
console.log(text.match(/<[Bb]>.*?<\/[Bb]>/g));
